### DISCUSSION TITLE
FR: Save and load hot keys when upgrading or testing new versions of FP

### DISCUSSION THREAD

**Discussion by Lotus49b** [freeplane/freeplane#1009] [Ideas]
It would be good if the user-defined hotkeys can be saved and reloaded into new versions of FP, or when using portable versions for testing. 
I tried saving the hotkeys in my "production" version, but when I attempt to load them from the portable version there's no option to open the hot key file.
![image](https://user-images.githubusercontent.com/11365744/218403341-f12ac09d-c957-4768-9425-aae099620bec.png)

Maybe even include hot key definitions in the Settings file? That would make it even simpler to migrate without forgetting something.

Thanks

**Comment by sifran-github** [comment]
You need to put the file that you saved (ends with ".properties") from the previous version in a directory called "accelerators"in the user directory, and you will be able to load it.

to get to the user directory: click on tools-> open user directory.
if there isn't a directory called "accelerators" there, you can create it, or save the current freeplane keys which will create it.

**Reply by Lotus49b** [reply]
Thanks!

It didn't work after following those steps (just realised I didn't create an "accelerators" directory as instructed - that might have been the reason it didn't work), but what did work was copying the contents of my saved hot key properties file into the accelerator.properties file in the user directory and then restarting FP.

All good now.

Saving the hot keys in the primary settings file will be more streamlined, but I guess there are so many requests with greater priority.

**Reply by sifran-github** [reply]
I think the reason that you didn't see a new file that you put there, is that you need to restart freeplane after you put a new file there.

**Reply by Lotus49b** [reply]
I did that too, but it didn't help.

Important thing is that it works now, and the solution is documented here now for anyone who wants to do the same thing.

Thanks very much for your help :)
